web_action_conditionable/README.rst

```diff
@@ -34,42 +36,45 @@ Not needed.
 Usage
 =====
 
-To use this module, you need to:
-
-* go to ...
+This module has no direct interface, it only adds functionality for custom views.
```

Comment:
Could provide info the functional need for this module.  It will be difficult for others to use this module if the functionality is not explained.

Reply:
Well, the functionality is explained a bit above: https://github.com/OCA/web/pull/845/files#diff-a6658a2cfba4db108729d91248a96babR9

I'm not sure what I could write in that section without repeating it.

Reply:
ok